docs/source/installation.rst

```diff
@@ -2,152 +2,133 @@ Installation
 ============
 
 Supported OS
-------------
+++++++++++++
 
-Currently, we only support ``MacOS`` and ``Linux`` operating systems. ``Windows``
-users have the possibility to install and use ``ivadomed`` via
-`Windows Subsystem for Linux (WSL) <https://docs.microsoft.com/en-us/windows/wsl/>`_. The steps below (about updating bashrc) are strongly recommended for MacOS users in the installation process but are optional for Linux users.
+    Currently, ``ivadomed`` supports GPU/CPU on ``Linux`` and ``Windows``, and CPU only on ``macOS`` and `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/>`_.
 
-Open your bash/zsh script file with editor on your computer.
+Step 1: Setup dedicated python environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    If you are using bash shell
-    ::
+    You can setup ``ivadomed`` using either Conda or Venv:
 
-        vim ~/.bashrc
+    .. tabs::
 
-    If you are using zsh shell
-    ::
-        
-        vim ~/.zshrc
+        .. tab:: Install via ``venv``
 
-Write in your .bashrc/.zshrc file with following line.
+            1. Setup Python Venv Virtual Environment.
 
-::
+                ``ivadomed`` requires Python >= 3.6 and <3.10.
 
-    export HDF5_USE_FILE_LOCKING='FALSE'
+                First, make sure that a compatible version of Python 3 is installed on your system by running:
 
-Save this change and restart your terminal to apply the change.
+                .. code::
 
-There are fundamentally two different approaches to install ``IvadoMed``:
+                   python3 --version
 
-1) Install via Conda
-    This is the easiest way for personal computers.
+                If your system's Python is not 3.7, 3.8, or 3.9 (or if you don't have Python 3 installed at all), please `install Python <https://realpython.com/installing-python/>`_ before continuing.
```

Comment:
Ack, sorry. RealPython is now prompting people to create accounts to read their pages.

![Screenshot from 2021-11-13 10-56-55](https://user-images.githubusercontent.com/16181459/141650466-67bc442c-230c-4c03-aeaa-754e8e7d0d8f.png)

Normally I like linking to RealPython because I find their pages very well-written, but this is enough of a nuisance that I think we should maybe link somewhere else.

```suggestion
                If your system's Python is not 3.7, 3.8, or 3.9 (or if you don't have Python 3 installed at all), please `install Python <https://wiki.python.org/moin/BeginnersGuide/Download/>`_ before continuing.
```

This new link is tied to the official Python wiki, so it should be less susceptible to that kind of thing.

Reply:
Funny I think maybe you visited the site wayyy too often? Just tested it in incognito and it was working? Did they cookie trap you lol. But I agree probably more legit linking the original repo. Going to make the change to official wiki.